templates/collections/blocks/time_period_explorer.html

```diff
@@ -5,11 +5,11 @@
 {% image value.page.specific.teaser_image fill-508x311 as teaser_image_large %}
 {% image value.page.specific.teaser_image fill-626x383 as teaser_image_extra_large %}
 
-<div class="card-group-promo--green">
+<li class="card-group-promo--green">
     <div class="card-group-promo__card">
         <div class="row">
             <div class="col-md-12 col-lg-7 col-xl-6">
-                <a href="{{ value.page.url }}" class="card-group-promo__card-link">
+                <a href="{{ value.page.url }}" class="card-group-promo__card-link" tabindex="-1">
```

Comment:
More of a comment than anything else, but we've removed the image link on the basis that the link is also presented in the text below and this avoids the user having to tab multiple times for each card.